neurostuff/requirements.txt

```diff
@@ -1,57 +1,20 @@
-aniso8601==8.0.0
-Babel==2.8.0
-blinker==1.4
-cachetools==4.1.1
-certifi==2020.6.20
-chardet==3.0.4
-click==7.1.2
-dnspython==2.0.0
-email-validator==1.1.1
+wheel
```

Comment:
I'm on board with not pinning to specific versions, but we should probably require minimum versions, as using old versions of some of these packages will definitely cause problems.